src/magic_folder/api_cli.py

```diff
@@ -401,7 +402,7 @@ def run_magic_folder_api_options(options):
 
         except MagicFolderApiError as e:
             # these kinds of errors should report via JSON from the endpoints
-            print(u"{}".format(e.body), file=options.stderr)
+            print(e, file=options.stderr)
```

Comment:
before this change, all output from magic-folder-api was json (idea is that's easier for other programs to integrate with).

Reply:
Not always. The next exception handler doesn't nor does `dump-state`. Also, before this PR, several of the commands would return HTML if given a folder that doesn't exist (it turns out `NoResource` returns HTML with the message passed to it).

That said, I do think having an option for JSON output may make sense, but I'm not I'm not sure that that it should be the default. That said, I don't feel strongly about this change (though something needs to be adjusted, as `MagicFolderApiError` has a parsed version, with `reason` broken out into it's own attribute, at the moment.

Reply:
I already proposed a change, just merge my pr-to-this-pr.

Reply:
Sure, `dump-state` doesn't do json by default (maybe it should?). But the commands that _do_ will now _sometimes_ return not-json, which is bad for integration.

Reply:
Bigger-picture here is that `magic-folder-api` is _supposed_ to be a fairly thin wrapper over the HTTP API and is _supposed_ to produce JSON output (e.g. https://github.com/LeastAuthority/magic-folder/issues/376 ). This change takes us further than that.

Reply:
I've reverted this to output json.